gamestonk_terminal/cryptocurrency/extra/gasnow_view.py

```diff
@@ -0,0 +1,27 @@
+import os
+from gamestonk_terminal.helper_funcs import export_data
+from gamestonk_terminal.cryptocurrency.extra.gasnow_model import get_gwei_fees
+
+
+def display_gwei_fees(export: str) -> None:
+    """Current gwei fees
+
+    Parameters
+    ----------
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+
+    df_fees = get_gwei_fees()
+
+    print("Current ETH gas fees (gwei):\n")
+    # for fee in fees:
+    #    print(f'\b {fee.label} ({fee.duration}): {fee.value} gwei')
+    print(df_fees.to_string(index=False))
```

Comment:
We should add there empty line under the table with gas prices.
![image](https://user-images.githubusercontent.com/275820/132358598-15ab8bb5-0c52-453c-8844-8b70f4fef30f.png)

`print(df_fees.to_string(index=False), "\n")`